### ISSUE TITLE
a11y: va-split

### ISSUE THREAD

**Issue by kushich** [epicmaxco/vuestic-ui#3268]
- [x] step 1: investigate a11y guides for this component;
- [ ] step 2: discuss with team;
- [ ] step 3: implement required functionality;
- [ ] step 4: add docs page section about a11y.

**Comment by kushich** [comment]
[ref](https://www.w3.org/WAI/ARIA/apg/patterns/windowsplitter/)
#### Keyboard navigation
- [ ] Left Arrow: Moves a vertical splitter to the left.
- [ ] Right Arrow: Moves a vertical splitter to the right.
- [ ] Up Arrow: Moves a horizontal splitter up.
- [ ] Down Arrow: Moves a horizontal splitter down.
- [ ] Enter: If the primary pane is not collapsed, collapses the pane (). If the pane is collapsed, restores the splitter to its previous position.
#### ARIA requirements 
- [ ] The element that serves as the focusable splitter has role [separator](https://w3c.github.io/aria/#separator).
- [ ] The separator element has the [aria-valuenow](https://w3c.github.io/aria/#aria-valuenow) property set to a decimal value representing the current position of the separator.
- [ ] The separator element has the [aria-valuemin](https://w3c.github.io/aria/#aria-valuemin) property set to a decimal value that represents the position where the primary pane has its minimum size. This is typically 0.
- [ ] The separator element has the [aria-valuemax](https://w3c.github.io/aria/#aria-valuemax) property set to a decimal value that represents the position where the primary pane has its maximum size. This is typically 100.
- [ ]  If the primary pane has a visible label, it is referenced by [aria-labelledby](https://w3c.github.io/aria/#aria-labelledby) on the separator element. Otherwise, the separator element has a label provided by [aria-label](https://w3c.github.io/aria/#aria-label).
- [ ]  The separator element has [aria-controls](https://w3c.github.io/aria/#aria-controls) referring to the primary pane.

_P.S. We should discuss pane priorities._

**Comment by asvae** [comment]
We can definitely implement focusability (which is missing right now) + keyboard navigation.

regarding aria tags - if that's possible for our implementation.
